pabi_procurement/models/purchase.py

```diff
@@ -105,6 +137,13 @@ def by_pass_approve(self, ids):
             po_rec.state = 'done'
         return True
 
+    @api.multi
+    def action_print_po(self):
```

Comment:
We don't need print button. You can remove it.
We want user to use menu action to print form.